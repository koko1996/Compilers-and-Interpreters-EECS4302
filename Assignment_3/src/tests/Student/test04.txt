var p // uninitialized
var q // uninitialized
var q	
verify p or q

