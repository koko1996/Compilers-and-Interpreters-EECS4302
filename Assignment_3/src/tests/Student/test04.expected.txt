Error: variable q already declared (3, 5)
